plugins/activity-resources/src/components/TxView.svelte

```diff
@@ -141,16 +143,49 @@
     edit = false
     props = getProps(props, edit)
   }
+
   function isMessageType (attr?: AnyAttribute): boolean {
     return attr?.type._class === core.class.TypeMarkup
   }
+
   function isAttachment (tx: TxCUD<Doc>): boolean {
     return tx.objectClass === attachment.class.Attachment && tx._class === core.class.TxCreateDoc
   }
+
   function isMention (_class?: Ref<Class<Doc>>): boolean {
     return _class === chunter.class.Backlink
   }
 
+  const reactionsQuery = createQuery()
```

Comment:
I think it should be part of TxCommentCreate and activity should not depend on chunter.

Reply:
It’s not possible to move all reactions content into `TxCommentCreate`, since we also need to add an action button. Defined an extension in the models and removed the activity dependency on chunter